### PR TITLE
增加了C/C++ Windows编程内容

### PR DESCRIPTION
**Pull request by Timothy-LiuXuefeng**
增加了C/C++ Windows编程内容

### CODE REVIEW

File: docs/languages/c_cpp_windows.md [hunk 1]
```diff
@@ -0,0 +1,549 @@
+
```
Comment (duskmoon314) [inline docs/languages/c_cpp_windows.md:1]: 建议添加标题字段，不然会显示c_cpp_windows
```suggestion
---
title: C/C++ Windows编程
---
```

**Comment by duskmoon314** [comment]
还需要在`sidebar.js`中加一下这个页面，不然除了url很难进入这个页面

**Comment by Timothy-LiuXuefeng** [comment]
Add title and change sidebar.js.

**Review by duskmoon314 — APPROVED** [review]
@all-contributors please add @Timothy-LiuXuefeng for doc

**Comment by duskmoon314** [comment]
@all-contributors please add @Timothy-LiuXuefeng for doc

**Comment by Timothy-Liuxf** [comment]
@all-contributors please add @Timothy-Liuxf for doc and remove @Timothy-LiuXuefeng

**Pull request merged**